src/transformers/models/poolformer/convert_poolformer_timm_to_pytorch.py

```diff
@@ -0,0 +1,214 @@
+# coding=utf-8
```

Comment:
Dumb question but why is the file named after timm if its a conversion from the original repo that doesn't look to have any link with timm?

Reply:
My bad! Will it be right to name it `convert_poolformer_original_to_pytorch.py`?

Reply:
Yes you can rename it like that.